Kickstarter-iOS/Views/Controllers/PledgeTableViewController.swift

```diff
@@ -129,6 +129,10 @@ extension PledgeTableViewController: PledgeDescriptionCellDelegate {
   internal func pledgeDescriptionCellDidPresentTrustAndSafety(_: PledgeDescriptionCell) {
     self.presentHelpWebViewController(with: .trust)
   }
+
+  internal func pledgeDescriptionCellDidTapRewardThumbnail(_: PledgeDescriptionCell) {
+    self.navigationController?.popViewController(animated: true)
```

Comment:
Yeah... good q. I mean that would allow us to test that if we call an input on the view model, it outputs that we should pop. It doesn't actually test that we called `popViewController`... so is it a meaningful test?

Reply:
I think it's still important that we pass everything through the view model so we have a complete account of all possible inputs and outputs for a given VC/VM combo, with tests. I think we also usually do this in other parts of the codebase, so if anything for consistency 😄

Reply:
Fair enough!